### PR TITLE
chore: clean up payments routes, documentation, payments utils

### PR DESCRIPTION
**Pull request by justynoh**
## Problem
This PR does some general clean-up, adding of documentation for payments-related routes and utils.

### CODE REVIEW

File: src/app/modules/form/admin-form/admin-form.controller.ts [hunk 1]
```diff
@@ -2704,92 +2694,3 @@ export const handleUpdateTwilio = [
...
- * @returns 200 with updated payments
- * @returns 403 when current user does not have permissions to update the payments
- * @returns 404 when form cannot be found
- * @returns 410 when updating the payments for an archived form
- * @returns 422 when user in session cannot be retrieved from the database
- * @returns 500 when database error occurs
- */
-export const _handleUpdatePayments: ControllerHandler<
-  { formId: string },
-  IEncryptedFormDocument['payments_field'] | ErrorDto,
```
Comment (justynoh) [inline src/app/modules/form/admin-form/admin-form.controller.ts:2723]: These functions moved to `admin-form.payments.controller` in tandem with moving payments routes to `admin-form.payments.routes`.

File: src/app/modules/payments/stripe.utils.ts [hunk 2]
```diff
@@ -17,14 +16,6 @@ import {
 
 const logger = createLoggerWithLabel(module)
 
-/**
- * Helper function to get redirect URI for OAuth callback on account linkage.
- */
-export const getRedirectUri = () =>
-  `${
-    config.isDev ? 'http://localhost:5001' : config.app.appUrl
```
Comment (justynoh) [inline src/app/modules/payments/stripe.utils.ts:25]: is only ever used once in the code, so this helper is unnecessary

File: src/app/routes/api/v3/admin/forms/admin-forms.form.routes.ts [hunk 3]
```diff
@@ -229,20 +229,3 @@ AdminFormsFormRouter.get(
...
- * @returns 403 when current user does not have permissions to update the payments
- * @returns 404 when form cannot be found
- * @returns 410 when updating the payments for an archived form
- * @returns 422 when user in session cannot be retrieved from the database
- * @returns 500 when database error occurs
- */
-AdminFormsFormRouter.put(
-  '/:formId([a-fA-F0-9]{24})/payments',
-  AdminFormController.handleUpdatePayments,
-)
```
Comment (justynoh) [inline src/app/routes/api/v3/admin/forms/admin-forms.form.routes.ts:248]: moved to `admin-forms.payments.routes`.

**Review by foochifa — APPROVED** [review]
LGTM thanks for this Justyn

**Pull request merged**